apps/web/src/server/lib/api-mapper.js

```diff
@@ -0,0 +1,16 @@
+import { pascalCase } from 'change-case';
+
+/**
+ * @constant {Function} replaceObjectKeysWithPascalCase
+ * @param {any} [object] - Object to convert
+ * @returns {object} The converted object.
+ * @description method returning object with keys converted to PascalCase
+ */
+export const replaceObjectKeysWithPascalCase = (object) =>
+	Object.fromEntries(
```

Comment:
Is the `object` parameter expected to not already be an object, or is this just to allow the function to take either an object or an array or map?

Reply:
My mistake, ignore me!